### PR TITLE
Synclist doc fixes

### PR DESCRIPTION
**Pull request by gurevichmark**
### The modification include

Rewrite of sync files sections of the docs and `xdcp` and `updatenode` man page changes related to synclist.

### CODE REVIEW

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst [hunk 1]
```diff
@@ -104,29 +104,24 @@ The noderange can have several formats. Following examples show that **/etc/host
...
 
 **EXECUTEALWAYS**
 
-The **EXECUTEALWAYS** clause is used to list all the postscripts you would like to run after the files are sync'd, whether or not any file is actually updated. The files in this list must be added to the list of files to rsync.  If noderange is used in the synclist for the file listed in the **EXECUTEALWAYS** clause, the script will only be exectuted on the nodes in that noderange.
+The **EXECUTEALWAYS** clause is used to list all the postsync scripts you would like to run after the files are synced, whether or not any file is actually updated. The files in this list must be added to the list of files to rsync.  If noderange is used in the synclist for the file listed in the **EXECUTEALWAYS** clause, the script will only be exectuted on the nodes in that noderange.
```
Comment (immarvin) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst]: `The files in this list must be added to the list of files to rsync` should better be `The files in this list must be added to the list of files to sync`

File: docs/source/guides/admin-guides/references/man1/xdcp.1.rst [hunk 2]
```diff
@@ -25,7 +25,7 @@ xdcp.1
 
 \ **xdcp**\  \ *computenoderange*\   [\ **-s**\  \ **-F**\  \ *rsync input file*\ ]
 
-\ **xdcp**\  [\ **-i**\  \ *path to install image*\ ] [\ **-F**\  \ *rsync input file*\ ]
+\ **xdcp**\  [\ **-i**\  \ *install image*\ ] [\ **-F**\  \ *rsync input file*\ ]
```
Comment (immarvin) [inline docs/source/guides/admin-guides/references/man1/xdcp.1.rst]: `[-r node_remote_shell]` is also supported here

**Comment by immarvin** [comment]
hi @gurevichmark , the manpage  of `[-r|--node-rcp]` option  for `xdcp` should be refined to be the similar with `[-r|--node-rcp]` option for `updatenode`  as following:

```
       [-r|--node-rcp [full_path_to_remote_copy_command]]
                 Specifies  the  full  path of the remote copy command used for syncing files to node targets, such as
                 "/usr/bin/rsync" or "/usr/bin/scp". If not specified, rsync will be used by default.

                 Notice: The synclist for "-r /usr/bin/scp" has some differences with "-r /usr/bin/rsync":

                 1) the ``EXECUTE`` clause is not supported in "-r /usr/bin/scp"

                 2) if the destination directory specified in synclist is an existing file on target node, "xdcp -r /usr/bin/scp -F<sync list>" will fail with ``scp: <destination directory>: Not a directory``

                 3) if the destination file specified in synclist is an existing directory on target node, "xdcp -r /usr/bin/scp -F<sync list>" will fail with ``scp: <destination file>: Is a directory``
```

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst [hunk 3]
```diff
@@ -104,29 +104,24 @@ The noderange can have several formats. Following examples show that **/etc/host
...
-
-Putting the filename.post in the **rsyncfile** to ``rsync`` to the node is required for hierarchical clusters. It is optional for non-hierarchical cluster.
-
 Advanced synclist file features
 '''''''''''''''''''''''''''''''
 
 **EXECUTE**
 
-The **EXECUTE** clause is used to list all the postscripts you would like to run after the files are sync'd, only if the file is updated. The files in this list must be added to the list of files to rsync. If noderange is used in the synclistfor the file listed in the **EXECUTE** clause, the script will only be exectuted on the nodes in that noderange.
+The **EXECUTE** clause is used to list all the postsync scripts (<filename>.post) you would like to run after the files are synced, only if the file <filename> is updated. For hierarchical clusters, the postsync files in this list must also be added to the list of files to sync. It is optional for non-hierarchical clusters. If noderange is used in the synclist for the file listed in the **EXECUTE** clause, the postsync script will only be executed on the nodes in that noderange.
```
Comment (immarvin) [inline docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst]: Need to mention that **EXECUTE** is not supported when the remote copy command is `scp`(option `-r|--node-rcp /usr/bin/scp` is specified)

Comment (gurevichmark) [reply docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst]: @immarvin What about **EXECUTEALWAYS** ? Is that supported if `-r|--node-rcp /usr/bin/scp` is specified ?

File: docs/source/guides/admin-guides/references/man1/xdcp.1.rst (hunk 2 above)
Comment (gurevichmark) [reply docs/source/guides/admin-guides/references/man1/xdcp.1.rst]: @immarvin Is it also supported for these 2 variations of **xdcp** ?
```
B<xdcp> I<noderange>  [B<-F> I<rsync input file>]

B<xdcp> I<computenoderange>  [B<-s> B<-F> I<rsync input file>]
```

**Comment by gurevichmark** [comment]
@immarvin I see this inconsistency between our man page for `xdcp` and `updatenode`:

`xdcp` man page says:
image

But `updatenode` says:

image

Do `xdcp` and `updatenode` have a different default, or is this an error in one of the man pages ?

File: docs/source/guides/admin-guides/references/man1/xdcp.1.rst (hunk 2 above)
Comment (immarvin) [reply docs/source/guides/admin-guides/references/man1/xdcp.1.rst]: yes, `[-r node_remote_shell]` is supported in this 2 variations

File: docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst (hunk 3 above)
Comment (immarvin) [reply docs/source/guides/admin-guides/manage_clusters/common/deployment/syncfile/syncfile_synclist_file.rst]: `EXECUTEALWAYS` is supported in both `-r` options

**Comment by immarvin** [comment]
hi, "`xdcp` man page" is incorrect. It should be modified to be consistent with `updatenode -F`

This is a 
```
[root@boston02 perl]# xdcp mid08tor03cn01 -v -T  -F /tmp/compute.synclist
TRACE:Default context is XCAT.
TRACE:Fanout Value is 64.
TRACE:Timeout Value is .
TRACE:Verifying remaining targets with pping command.
 TRACE: Executing Command:/bin/sh -c /tmp/rsync_mid08tor03cn01
 ...

[root@boston02 perl]# xdcp mid08tor03cn01 -v -T  -r /usr/bin/scp -F /tmp/compute.synclist
TRACE:Default context is XCAT.
TRACE:Fanout Value is 64.
TRACE:Timeout Value is .
TRACE:Verifying remaining targets with pping command.
 TRACE: Executing Command:/bin/sh -c /tmp/scp_mid08tor03cn01
 ...

 [root@boston02 perl]# xdcp mid08tor03cn01 -v -T  -r /usr/bin/rsync -F /tmp/compute.synclist
TRACE:Default context is XCAT.
TRACE:Fanout Value is 64.
TRACE:Timeout Value is .
TRACE:Verifying remaining targets with pping command.
 TRACE: Executing Command:/bin/sh -c /tmp/rsync_mid08tor03cn01
 ...
```

**Comment by gurevichmark** [comment]
New commit added to fix review suggestions.

**Comment by immarvin** [comment]
agree to merge, thanks @gurevichmark

**Pull request merged**